packages/api/src/auth/decoders/auth0.ts

```diff
@@ -33,44 +33,33 @@ export const verifyAuth0Token = (
       )
     }
 
-    if (
```

Comment:
@peterp we thought it best to maintain the same behaviour between dev and production for token validation. If there wasn't another reason you had it this way, are you happy with this change?

Reply:
For test: yeah we can keep it there I guess. Would you expect jest to run an actual test calling this? I would imagine these decoders would be mocked (but I may be wrong).

But for dev, we should atleast do what we're doing on netlify, so the expirng tokens are a  _visible_ problem to the user, so they can fix.

Reply:
Ok I will leave as is then.

Reply:
@peterp, just need the 👍 from you :)